Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -1999,6 +2048,103 @@ def fetch_incidents(client: Client, last_run: Dict[str, Any], params: Dict[str,
     return incidents, ids_to_insert, updated_last_run_time
 
 
+def access_key_create_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    access_key = client.access_key_creation(args)
+    transform_access_keys = {
+        'Id': access_key.get('id'),
+        'Secret Key': access_key.get('secretKey'),
+    }
+
+    markdown_table = tableToMarkdown('PrismaCloud Access Key Creation', transform_access_keys, headers=['Id', 'Secret Key'])
```

Comment:
If the response contains only id and secretKey, let's use headerTranform of tableToMarkdown instead of transform_access_keys. look for examples in our repo.